doc/radosgw/cloud-transition.rst

```diff
@@ -50,6 +50,8 @@ Cloud Storage Class Tier Configuration
                 "dest_id": <dest_id> } ... ],
     "location_constraint": <location-constraint>,
     "target_path": <target_path>,
+    "target_by_bucket": <true | false>,
```

Comment:
It might help to add an example using these options.

Reply:
I have included that.